packages/driver/cypress/e2e/commands/actions/scroll.cy.js

```diff
@@ -27,6 +27,10 @@ describe('src/cy/commands/actions/scroll', () => {
 
       this.scrollBoth.scrollTop = 0
       this.scrollBoth.scrollLeft = 0
+
+      // width/height of scrollable container - width of parent viewport (minux scrollbars) / 2 to get the center
+      // browsers round up the pixel value so we need to round it
+      this.halfScrollPixels = Math.round((500 - 100) / 2)
```

Comment:
just moving this calc to one place with some explanation because it took me a while to figure out these values from thin air.